webextensions/api/runtime.json

```diff
@@ -50,11 +88,72 @@
               }
             }
           },
-          "tlsChannelId": {
+          "id": {
             "__compat": {
               "support": {
                 "chrome": {
-                  "version_added": "32"
+                  "version_added": "26"
+                },
+                "edge": "mirror",
+                "firefox": {
+                  "version_added": "45"
+                },
+                "firefox_android": {
+                  "version_added": "48"
+                },
+                "opera": "mirror",
+                "safari": {
+                  "version_added": "14"
+                },
+                "safari_ios": {
+                  "version_added": "15"
+                }
+              }
+            }
+          },
+          "nativeApplication": {
+            "__compat": {
+              "support": {
+                "chrome": {
+                  "version_added": "74"
+                },
+                "edge": "mirror",
+                "firefox": {
+                  "version_added": false
+                },
+                "firefox_android": "mirror",
+                "opera": "mirror",
+                "safari": {
+                  "version_added": false
+                },
+                "safari_ios": "mirror"
+              }
+            }
+          },
```

Comment:
(suggested edit as a follow-up to the thread at https://github.com/mdn/browser-compat-data/pull/20970/files#r1361469907 )

```suggestion
```

Reply:
Thanks @Rob--W i'll pull the documentation from the parallel content request too